package.json

```diff
@@ -53,7 +53,7 @@
     "prettier": "prettier -c --write \"**/**\"",
     "pull": "git pull",
     "release": "semantic-release",
-    "start": "next start",
+    "start": "next start -p 3210",
```

Comment:
I did not mean to commit this change

Reply:
I think it might be a good change for user who want to build from stratch.Because the port will be the same as docker. Maybe you can seperate this update to another PR and we can merge it quickly.